src/main/java/org/betonquest/betonquest/events/DropItemEvent.java

```diff
@@ -112,42 +121,48 @@ public void onPlayerJoin(final PlayerJoinEvent event) {
 
     @EventHandler(ignoreCancelled = true)
     public void onMergingItem(final ItemMergeEvent event) {
-        for (final Entity item : entityPlayerMap.keySet()) {
-            if (event.getEntity().equals(item)) {
-                event.setCancelled(true);
-            }
-        }
-        for (final Entity item : indestructibleItem) {
-            if (event.getEntity().equals(item)) {
-                event.setCancelled(true);
+        //TODO: Define default behavior: Shouldn't private items always be protected from being converted into public
+        //      items by being merged with another players private items?
```

Comment:
previously, When I try not cancelling the event, the item will get merge with PUBLIC / VISIBLE dropped item. Thats why I cancelling all the item in entityPlayerMap and indestructibleItem.

Reply:
Then you can rollback my changes and just use 
```JAVA
 if (event.getEntity().equals(item)) {
     event.setCancelled(true);
  }
```